packages/radix-vue/src/RangeCalendar/story/RangeCalendarChromatic.story.vue

```diff
@@ -47,8 +49,16 @@ const placeholder = ref(new CalendarDate(2024, 4, 1)) as Ref<CalendarDate>
       <RangeCalendar :default-value="defaultValue" prevent-deselect />
     </Variant>
 
+    <Variant title="Start value">
+      <RangeCalendar :default-value="defaultValue" :start-value="startValue" />
```

Comment:
removing `defaultValue`, and allow `startValue` to take action, causing the placeholder month to be different from the `startValue`.

Maybe let's show the same month as the `startValue`? That behave more "natural"

![image](https://github.com/radix-vue/radix-vue/assets/59365435/7d35a686-29f4-4b17-8b26-58b54dd54611)